### PR TITLE
Linked dashboard to backend

### PR DESCRIPTION
**Pull request by hanyuone**
I've stared at this code for way too long and the Dashboard component is really long for my liking - if someone could help clean it up that would be great haha

### CODE REVIEW

File: Cms_frontend/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -9,69 +9,117 @@ import NewDialogue from 'src/components/NewDialogue/NewDialogue';
...
+    const root = await getFolder();
+    setDir([root]);
+    setLoading(false);
+  }
+
+  const getCurrentID = () => {
+    return dir[dir.length - 1].id;
+  }
+
+  useEffect(() => {
```
Comment (fafnirZ) [inline Cms_frontend/src/pages/Dashboard.tsx:76]: can probably add a comment to make the use effects stand out more

File: Cms_frontend/src/data/dummy_structure.json [hunk 2]
```diff
@@ -1,46 +1,55 @@
 {
     "root": [
         {
+            "id": 0,
```
Comment (fafnirZ) [inline Cms_frontend/src/data/dummy_structure.json:4]: why is this file still necessary if theres a backend?

Comment (hanyuone) [reply Cms_frontend/src/data/dummy_structure.json:4]: Might as well keep around for test purposes? Will delete if it's not necessary though

Comment (fafnirZ) [reply Cms_frontend/src/data/dummy_structure.json:4]: ahhh okay, I just thought your logic still implemented it,  but could probably move it in a test folder then so it isnt confusing

**Review by fafnirZ — APPROVED** [review]

File: Cms_frontend/src/data/dummy_structure.json (hunk 2 above)
Comment (omeh-a) [reply Cms_frontend/src/data/dummy_structure.json:4]: It's best to remove this. We can interact directly with the DB for testing cases.

File: Cms_frontend/src/pages/Dashboard.tsx [hunk 3]
```diff
@@ -9,69 +9,117 @@ import NewDialogue from 'src/components/NewDialogue/NewDialogue';
...
 
+  const toFileFormat = (json: any): FileFormat => {
+    return {
+      id: json.EntityID,
+      filename: json.EntityName,
+      isDocument: json.IsDocument
+    };
+  }
+
+  const getFolder = async (id?: number) => {
```
Comment (omeh-a) [inline Cms_frontend/src/pages/Dashboard.tsx:39]: Might be worth adding a brief comment to indicate what these funcs are for

**Review by omeh-a — APPROVED** [review]
